packages/astro/test/units/test-utils.js

```diff
@@ -62,6 +62,9 @@ class VirtualVolume extends Volume {
 class VirtualVolumeWithFallback extends VirtualVolume {
 	// Fallback to the real fs
 	readFile(p, ...args) {
+		if(p?.toString().includes('astro:dev-module-loader')) {
+			throw new Error('This module can\'t be loaded threw here. blah');
```

Comment:
Not sure how likely that error is, but if it's real, a good error message would be better so we don't eventually have to explain to a new contributor why that error happens, ha